src/SFA.DAS.FAT.Web/Views/Courses/CourseProviderDetail.cshtml

```diff
@@ -42,26 +42,59 @@
                         <div class="govuk-summary-list__row">
                             <dt class="govuk-summary-list__key">Email</dt>
                             <dd class="govuk-summary-list__value"><a class="govuk-link govuk-link--no-visited-state" href="mailto:@Model.Email">@Model.Email</a></dd>
-                        </div>    
+                        </div>
+
                     }
                     @if (!string.IsNullOrEmpty(Model.Phone))
                     {
                         <div class="govuk-summary-list__row">
                             <dt class="govuk-summary-list__key">Telephone</dt>
                             <dd class="govuk-summary-list__value">@Model.Phone</dd>
-                        </div>    
+                        </div>
+
                     }
                     @if (!string.IsNullOrEmpty(Model.Website))
                     {
                         <div class="govuk-summary-list__row">
                             <dt class="govuk-summary-list__key">Website</dt>
                             <dd class="govuk-summary-list__value"><a class="govuk-link govuk-link--no-visited-state" target="_blank" rel="noreferrer noopener" href="@Model.Website">@Model.Website</a></dd>
-                        </div>    
+                        </div>
+
                     }
-                    
+
                 </dl>
 
             </div>
+            @if (Model.AdditionalCourses.Total > 0)
+            {
+                <details class="govuk-details govuk-!-margin-top-7" data-module="govuk-details">
+                    <summary class="govuk-details__summary">
+                        <span class="govuk-details__summary-text govuk-!-font-size">
+                            @(Model.AdditionalCourses.Total == 1?
+                            "View the 1 other course offered by this provider" :
+                            $"View all {Model.AdditionalCourses.Total} courses offered by this provider")
+                        </span>
+                    </summary>
+                    <div class="govuk-details__text">
+                        <p class="govuk-body ">Select a course to view locations, training options and contact details.</p>
+                        <table class="govuk-table">
+                            <tbody class="govuk-table__body">
+                                @foreach (var course in Model.AdditionalCourses.Courses)
+                                {
+                                    <tr class="govuk-table__row">
+                                        <td class="govuk-table__cell">
+                                            <a class="govuk-link govuk-link--no-visited-state"
+                                               asp-route="@RouteNames.CourseProviderDetails"
+                                               asp-route-id="@course.Id"
+                                               asp-route-providerId="@Model.ProviderId">@course.TitleAndLevel</a>
```

Comment:
This is going to the wrong action, it should be:

`<a class="govuk-link govuk-link--no-visited-state"
                                               asp-route="@RouteNames.CourseDetails"
                                               asp-route-id="@course.Id">@course.TitleAndLevel</a>`

Reply:
Ignore!